extension/server/src/server.ts

```diff
@@ -135,7 +135,7 @@ parser.setIncludeFileFetch(async (stringUri: string, includeString: string) => {
 		// This is likely bad since you can include across file systems.
 
 		const hasQuotes = (includeString.startsWith(`'`) && includeString.endsWith(`'`)) || (includeString.startsWith(`"`) && includeString.endsWith(`"`))
-		const isUnixPath = hasQuotes || includeString.includes(`/`);
+		const isUnixPath = hasQuotes || (includeString.includes(`/`) && !includeString.includes(`,`));
```

Comment:
I'm not too happy about the new test for `,`:

```javascript
const isUnixPath = hasQuotes || (includeString.includes(`/`) && !includeString.includes(`,`));
```

Considering a `,` in a filename a QSYS member does not quite follow the compiler behaviour - and someone may be burnt by this. Ideally we should follow [the rules](https://www.ibm.com/docs/en/i/7.3?topic=files-include) as closely as possible and search in both filesystems, unless the name can only exist in one filesystem.

I don't think we should worry about performance too much. Most times the include files will be in the expected place and will be found quickly - and if they are in another filesystem, a little wait is okay. Better right than quick! 😆

Reply:
> Considering a , in a filename a QSYS member does not quite follow the compiler behaviour - and someone may be burnt by this.

You are likely right, but I am happy to say that vscode-rpgle won't support every method of including sources from the get go. For example, there is no plan for me to add support for this as of right now: `/COPY "A/b",mymbr`

I also really don't want to add code to search members first and then fall back to IFS files. This is a personal preference because I am worried about performance (sorry!) and I remember when I tried to write this a few years ago it didn't go so well.

I would like our code to determine if this is an IFS include OR a member include and then try to bring them in. I will try and think of another method instead of checking for `,` 😆